### PR TITLE
Ec2 instance using IAC

### PR DESCRIPTION
**Pull request by jayvardhan2111**

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+output "public_ip" {
+  value = aws_instance.public.public_ip
+
+}
+
+output "private_ip" {
+  value = aws_instance.private.private_ip
+}
```
Comment (geoffrey1330) [inline outputs.tf]: Add newline at the end of file.

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: outputs.tf (hunk 1 above)
Comment (jayvardhan2111) [reply outputs.tf]: I have added the new line. 
Can I know why we add new line i.e what errors may occur ?

Comment (geoffrey1330) [reply outputs.tf]: Adding newline is a good code practice and makes your code looks neat and it is always flag by github to be added when your PR is being reviewed.

Comment (geoffrey1330) [reply outputs.tf]: Adding newline is a good code practice and makes your code looks neat and it is always flag by github to be added when your PR is being reviewed.

**Review by geoffrey1330 — APPROVED** [review]

File: outputs.tf (hunk 1 above)
Comment (jayvardhan2111) [reply outputs.tf]: okay

**Pull request merged**